src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -51,8 +66,8 @@ class TestCaseDisplayService(private val project: Project) {
             checkbox.isSelected = true
             testCasePanel.add(checkbox, BorderLayout.WEST)
 
-            val code = JavaCodeFragmentFactory.getInstance(project)
-                .createExpressionCodeFragment(testCode, null, null, true)
+            val code =
+                JavaCodeFragmentFactory.getInstance(project).createExpressionCodeFragment(testCode, null, null, true)
```

Comment:
That looks a bit ugly. Could you change it back to how it was so that it's consistent with the rest of the project?

Reply:
Yeah, I wanted to ask about this also since I wasn't sure but the ctrl+alt+l changed it. Should be fixed now.